.eslintrc.json

```
@@ -1,7 +1,7 @@
 {
   "parser": "babel-eslint",
   "extends": [
-    "skyscanner",
+    "@skyscanner/skyscanner",
```

Whilst this works due to eslint magic we should change this to follow our update docs for this to be 

```suggestion
    "@skyscanner/eslint-config-skyscanner",
```

https://github.com/Skyscanner/eslint-config-skyscanner/#installation